packages/@aws-cdk/cloudformation-include/test/test-templates/fn-sub-valid-string.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": "some-bucket${!AWS::AccountId}2453${ ! AWS::Region}1-1${!Immediate}234"
```

Comment:
This test is weird... I don't know whether the escaping should work here, or not (because of the whitespace).

Can you please move these escaping edge-cases to their own test(s), and just leave this as a simple test for the positive case (no escaping)?